src/main/java/com/alkemy/ong/controller/UserController.java

```diff
@@ -35,6 +36,10 @@
         List<UserDTO> users = userService.getAllUser();
         return ResponseEntity.ok().body(users);
     }
+    @GetMapping("auth/me")
```

Comment:
is this method in the UserAuthController? should the route be only "/me"? I am not sure if it is the same

Reply:
this method shows the data after authenticating and the endpoints are public of UserAuthController

Reply:
got it thanks!